wolfcrypt/src/sp_cortexm.c

```diff
@@ -190,500 +190,500 @@ SP_NOINLINE static void sp_2048_mul_8(sp_digit* r, const sp_digit* a,
         const sp_digit* b)
 {
     sp_digit tmp[8];
-    __asm__ __volatile__ (
+    asm volatile (
```

Comment:
Is this change for `asm` and `volatile` okay for all compilers? Perhaps we should have macros for these and detect the `__IAR_SYSTEMS_ICC__` case?

Reply:
Changed as requested.